pybio/torch/transformations/base.py

```diff
@@ -1,42 +1,39 @@
 # TODO would be nice to have something similar to inferno's batch_function etc.
 # functionality, but more flexible w.r.t implementing different dimensions.
+from typing import List, Optional, Sequence, Tuple
 
+import torch
 
-class IndependentTransformation:
+import pybio.core.transformations
```

Comment:
Do we need a transformation base class in `pybio.core`?
Right now this feels like it's just producing more boilerplate code to me.
**Edit:** It seems like you have moved the implementation of the `__call__` method there, but I am not sure if this is the good idea,
because it's (implicitly) imposing the specific transformation mechanism on everyone who would implement the `pybio` API.

Reply:
I think it would be great if torchbio's transformations behave the same way pybio ones do, therefore I would inherit from the pybio base transformation. It's there anyway for the framework agnostic transformations.

there should not be any `__call__` methods left for transformations... ?

Reply:
oh, sorry I confused Transformation with BaseTransformation... yeah.. let's get rid of the latter, it seems to cause confusions^^